recipe/meta.yaml

```diff
@@ -644,8 +658,168 @@ outputs:
         - numpy                                  # [build_platform != target_platform]
         - cmake
         - ninja
+      host:
+        - {{ pin_subpackage("libarrow", exact=True) }}
+        - {{ pin_subpackage("libarrow-acero", exact=True) }}
```

Comment:
Can you add a comment on why there are listed in `host` but not in `run`? It looks like you did this intentionally, but it would be better if we recorded this decision.

Reply:
@raulcd, would you like to formulate a response to this request? Otherwise I can probably come up with something.

Reply:
I am currently traveling. I will be able to get back to this around Friday, I am happy if you can come up with something before that, otherwise I'll do it then.

Reply:
I've added the comment @xhochy let me know if that makes sense and are you ok with me merging this